### PR TITLE
WFLY-19534 Fix grammar in JaxrsLogger#typeNameNotAnEjbView and EjbLogger#invalidEjbComponent + across the code base

### PR DESCRIPTION
**Pull request by rhusar**
Resolves
https://issues.redhat.com/browse/WFLY-19534

### CODE REVIEW

File: docs/src/main/asciidoc/_developer-guide/ejb3/Jakarta_Enterprise_Beans_Database_Persistent_Timers.adoc [hunk 1]
```diff
@@ -125,7 +125,7 @@ out of sync. Some application may find the `refresh-interval` configuration not
 need to programmatically refresh timers. This can be done with Jakarta Interceptors configured for those business methods
 that need this capability, as illustrated in the following steps:
 
-* Implement an Jakarta Interceptors that enables `wildfly.ejb.timer.refresh.enabled` to true. For example,
+* Implement a Jakarta Interceptors that enables `wildfly.ejb.timer.refresh.enabled` to true. For example,
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_developer-guide/ejb3/Jakarta_Enterprise_Beans_Database_Persistent_Timers.adoc:128]: Your change is fine and I don't want to increase your scope of work, so feel free to ignore...

Either
"a Jakarta Interceptors interceptor"

or 

"an interceptor"

File: docs/src/main/asciidoc/_developer-guide/ejb3/jboss-ejb3.xml_Reference.adoc [hunk 2]
```diff
@@ -78,7 +78,7 @@ or to all beans in the deployment by using `*` as the `ejb-name`.
 ==== The security namespace urn:security
 
 This allows you to set the security domain and the run-as principal for
-an Jakarta Enterprise Beans.
+a Jakarta Enterprise Beans.
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_developer-guide/ejb3/jboss-ejb3.xml_Reference.adoc:81]: probably "Bean", unless this is about something that's scoped to > 1 bean, in which case no "an" or "a"

File: docs/src/main/asciidoc/_extending-wildfly/WildFly_JNDI_Implementation.adoc [hunk 3]
```diff
@@ -37,9 +37,9 @@ multiple naming stores in every WildFly instance, serving different JNDI
 namespaces:
 
 * java:comp - the standard EE namespace for entries scoped to a specific
-component, such as an Jakarta Enterprise Beans
+component, such as a Jakarta Enterprise Beans
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_extending-wildfly/WildFly_JNDI_Implementation.adoc:40]: "Bean"

File: docs/src/main/asciidoc/_testsuite/Pre-requisites_-_test_quality_standards.adoc [hunk 4]
```diff
@@ -59,7 +59,7 @@ subsystem (e.g org.jboss.as.test.integration.ejb.async). When a test +
 impacts multiple subsystems this is a bit of a judgement call, but in +
 general the tests should go into the package of +
 the spec that defines the functionality (e.g. Jakarta Contexts and Dependency Injection based constructor +
-injection into an Jakarta Enterprise Beans, even though this involves Jakarta Contexts and Dependency Injection and Jakarta Enterprise Beans, +
+injection into a Jakarta Enterprise Beans, even though this involves Jakarta Contexts and Dependency Injection and Jakarta Enterprise Beans, +
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_testsuite/Pre-requisites_-_test_quality_standards.adoc:62]: "Bean"

File: ejb3/src/main/java/org/jboss/as/ejb3/component/ContainerInterceptorMethodInterceptorFactory.java [hunk 5]
```diff
@@ -17,7 +17,7 @@
 
 /**
  * An {@link InterceptorFactory} responsible for creating {@link Interceptor} instance corresponding to a <code>container-interceptor</code>
- * applicable for an Jakarta Enterprise Beans
+ * applicable for a Jakarta Enterprise Beans
```
Comment (bstansberry) [inline ejb3/src/main/java/org/jboss/as/ejb3/component/ContainerInterceptorMethodInterceptorFactory.java:20]: "Bean"

File: ejb3/src/main/java/org/jboss/as/ejb3/component/EJBComponentUnavailableException.java [hunk 6]
```diff
@@ -9,7 +9,7 @@
 
 /**
  * An exception which can be used to indicate that a particular Jakarta Enterprise Beans component is (no longer) available for handling invocations.
- * This typically is thrown when an Jakarta Enterprise Beans are invoked
+ * This typically is thrown when a Jakarta Enterprise Beans are invoked
```
Comment (bstansberry) [inline ejb3/src/main/java/org/jboss/as/ejb3/component/EJBComponentUnavailableException.java:12]: "Bean is"

**Comment by rhusar** [comment]
@bstansberry Regarding the singular form, the "Jakarta Enterprise Beans" is referencing the spec name verbatim, so should we go with "Jakarta Enterprise Beans bean" or is that over the top?

File: ejb3/src/main/java/org/jboss/as/ejb3/component/EJBValidationConfigurator.java [hunk 7]
```diff
@@ -21,7 +21,7 @@
 
 /**
  *
- * Configurator that validates than an Jakarta Enterprise Beans class does not validate the Jakarta Enterprise Beans specification
+ * Configurator that validates than a Jakarta Enterprise Beans class does not validate the Jakarta Enterprise Beans specification
```
Comment (bstansberry) [inline ejb3/src/main/java/org/jboss/as/ejb3/component/EJBValidationConfigurator.java:24]: s/than/that

also, the 2nd "validate" should be "violate"

File: ejb3/src/main/java/org/jboss/as/ejb3/component/interceptors/GetHomeInterceptorFactory.java [hunk 8]
```diff
@@ -12,7 +12,7 @@
 import org.jboss.msc.value.InjectedValue;
 
 /**
- * Interceptor that can return a home interface for an Jakarta Enterprise Beans
+ * Interceptor that can return a home interface for a Jakarta Enterprise Beans
```
Comment (bstansberry) [inline ejb3/src/main/java/org/jboss/as/ejb3/component/interceptors/GetHomeInterceptorFactory.java:15]: "Bean"

File: ejb3/src/main/java/org/jboss/as/ejb3/component/stateful/StatefulIdentityInterceptor.java [hunk 9]
```diff
@@ -32,7 +32,7 @@ public Object processInvocation(final InterceptorContext context) throws Excepti
                 final Class<?> proxyType = componentView.getProxyClass();
                 final SessionID sessionId = context.getPrivateData(SessionID.class);
                 if( proxyType.isAssignableFrom(other.getClass())) {
-                    //now we know that this is an Jakarta Enterprise Beans for the correct component view
+                    //now we know that this is a Jakarta Enterprise Beans for the correct component view
```
Comment (bstansberry) [inline ejb3/src/main/java/org/jboss/as/ejb3/component/stateful/StatefulIdentityInterceptor.java:35]: "Bean"

File: ejb3/src/main/java/org/jboss/as/ejb3/deployment/EjbDeploymentInformation.java [hunk 10]
```diff
@@ -19,7 +19,7 @@
 import java.util.Set;
 
 /**
- * Runtime information about an Jakarta Enterprise Beans in a module
+ * Runtime information about a Jakarta Enterprise Beans in a module
```
Comment (bstansberry) [inline ejb3/src/main/java/org/jboss/as/ejb3/deployment/EjbDeploymentInformation.java:22]: "Bean"

File: ejb3/src/main/java/org/jboss/as/ejb3/deployment/processors/dd/SecurityRoleRefDDProcessor.java [hunk 11]
```diff
@@ -17,7 +17,7 @@
 import org.jboss.metadata.javaee.spec.SecurityRoleRefsMetaData;
 
 /**
- * Sets up the {@link EJBComponentDescription} with the &lt;security-role-ref&gt;s declared for an Jakarta Enterprise Beans
+ * Sets up the {@link EJBComponentDescription} with the &lt;security-role-ref&gt;s declared for a Jakarta Enterprise Beans
```
Comment (bstansberry) [inline ejb3/src/main/java/org/jboss/as/ejb3/deployment/processors/dd/SecurityRoleRefDDProcessor.java:20]: "Bean"

File: ejb3/src/main/java/org/jboss/as/ejb3/deployment/processors/merging/AbstractPoolMergingProcessor.java [hunk 12]
```diff
@@ -23,7 +23,7 @@
 
 /**
  * Sets up the Jakarta Enterprise Beans component description with the pool name configured via the {@link org.jboss.ejb3.annotation.Pool}
- * annotation and/or the deployment descriptor, for an Jakarta Enterprise Beans
+ * annotation and/or the deployment descriptor, for a Jakarta Enterprise Beans
```
Comment (bstansberry) [inline ejb3/src/main/java/org/jboss/as/ejb3/deployment/processors/merging/AbstractPoolMergingProcessor.java:26]: "Bean"

File: ejb3/src/main/java/org/jboss/as/ejb3/deployment/processors/merging/MethodPermissionsMergingProcessor.java [hunk 13]
```diff
@@ -39,7 +39,7 @@
 /**
  * Handles the {@link jakarta.annotation.security.RolesAllowed} {@link DenyAll} {@link jakarta.annotation.security.PermitAll} annotations
  * <p/>
- * Also processes the &lt;method-permission&gt; elements of an Jakarta Enterprise Beans and sets up appropriate security permissions on the Jakarta Enterprise Beans.
+ * Also processes the &lt;method-permission&gt; elements of a Jakarta Enterprise Beans and sets up appropriate security permissions on the Jakarta Enterprise Beans.
```
Comment (bstansberry) [inline ejb3/src/main/java/org/jboss/as/ejb3/deployment/processors/merging/MethodPermissionsMergingProcessor.java:42]: "Bean"

File: ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java [hunk 14]
```diff
@@ -125,7 +125,7 @@ public interface EjbLogger extends BasicLogger {
 //    void cacheRemoveFailed(Object id);
 
 //    /**
-//     * Logs a warning message indicating an Jakarta Enterprise Beans for the specific id could not be found
+//     * Logs a warning message indicating a Jakarta Enterprise Beans for the specific id could not be found
```
Comment (bstansberry) [inline ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java:128]: "Bean"

File: ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java [hunk 15]
```diff
@@ -365,15 +365,15 @@ public interface EjbLogger extends BasicLogger {
     void invocationFailed(String component, Method method, @Cause Throwable t);
 
     /**
-     * Logs an error message indicating that an Jakarta Enterprise Beans client proxy could not be swapped out in a RMI invocation
+     * Logs an error message indicating that a Jakarta Enterprise Beans client proxy could not be swapped out in a RMI invocation
```
Comment (bstansberry) [inline ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java:368]: "Bean"

File: ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java [hunk 16]
```diff
@@ -365,15 +365,15 @@ public interface EjbLogger extends BasicLogger {
...
      */
     @LogMessage(level = WARN)
     @Message(id = 35, value = "Could not find Jakarta Enterprise Beans for locator %s, Jakarta Enterprise Beans client proxy will not be replaced")
     void couldNotFindEjbForLocatorIIOP(EJBLocator<?> locator);
 
 
     /**
-     * Logs an error message indicating that an Jakarta Enterprise Beans client proxy could not be swapped out in a RMI invocation
+     * Logs an error message indicating that a Jakarta Enterprise Beans client proxy could not be swapped out in a RMI invocation
```
Comment (bstansberry) [inline ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java:376]: "Bean"

File: ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java [hunk 17]
```diff
@@ -2573,12 +2573,12 @@ public interface EjbLogger extends BasicLogger {
     DeploymentUnitProcessingException componentClassHasMultipleTimeoutAnnotations(Class<?> componentClass);
 
     /**
-     * Creates an exception indicating the current component is not an Jakarta Enterprise Beans.
+     * Creates an exception indicating the current component is not a Jakarta Enterprise Beans.
```
Comment (bstansberry) [inline ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java:2576]: "Bean"

File: ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java [hunk 18]
```diff
@@ -2573,12 +2573,12 @@ public interface EjbLogger extends BasicLogger {
...
      *
      * @param component the component.
      * @return an {@link IllegalStateException} for the error.
      */
-    @Message(id = 373, value = "Current component is not an Jakarta Enterprise Beans %s")
+    @Message(id = 373, value = "Current component is not a Jakarta Enterprise Beans %s")
```
Comment (bstansberry) [inline ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java:2581]: "Bean"

File: ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java [hunk 19]
```diff
@@ -2658,7 +2658,7 @@ public interface EjbLogger extends BasicLogger {
     IllegalArgumentException stringParamCannotBeNullOrEmpty(final String paramName);
 
     /**
-     * Exception that is thrown when invoking remove while an Jakarta Enterprise Beans is in a transaction
+     * Exception that is thrown when invoking remove while a Jakarta Enterprise Beans is in a transaction
```
Comment (bstansberry) [inline ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java:2661]: "Bean"

File: ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java [hunk 20]
```diff
@@ -2638,7 +2638,7 @@ public interface EjbLogger extends BasicLogger {
     IllegalStateException unknownMessageListenerType(String resourceAdapterName, String messageListenerType);
 
     /**
-     * Thrown when an Jakarta Enterprise Beans 2 Jakarta Enterprise Beans does not implement a method on an Jakarta Enterprise Beans 2
+     * Thrown when a Jakarta Enterprise Beans 2 Jakarta Enterprise Beans does not implement a method on a Jakarta Enterprise Beans 2
```
Comment (bstansberry) [inline ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java:2641]: this is some strange alien language ;)

File: ejb3/src/main/java/org/jboss/as/ejb3/security/EJBMethodSecurityAttribute.java [hunk 21]
```diff
@@ -11,7 +11,7 @@
 import java.util.Set;
 
 /**
- * Holds security metadata of a method corresponding to an Jakarta Enterprise Beans's view.
+ * Holds security metadata of a method corresponding to a Jakarta Enterprise Beans's view.
```
Comment (bstansberry) [inline ejb3/src/main/java/org/jboss/as/ejb3/security/EJBMethodSecurityAttribute.java:14]: "Bean's"

File: ejb3/src/main/java/org/jboss/as/ejb3/security/EjbJaccConfig.java [hunk 22]
```diff
@@ -11,7 +11,7 @@
 import java.util.Map;
 
 /**
- * Class that stores the JACC config for an Jakarta Enterprise Beans before being consumed by the JACC service.
+ * Class that stores the JACC config for a Jakarta Enterprise Beans before being consumed by the JACC service.
```
Comment (bstansberry) [inline ejb3/src/main/java/org/jboss/as/ejb3/security/EjbJaccConfig.java:14]: "Bean"

File: jaxrs/src/main/java/org/jboss/as/jaxrs/logging/JaxrsLogger.java [hunk 23]
```diff
@@ -133,7 +132,7 @@ public interface JaxrsLogger extends BasicLogger {
      * @param ejbName The ejb
      * @return  the exception
      */
-    @Message(id = 10, value = "Jakarta RESTful Web Services resource %s does not correspond to a view on the Jakarta Enterprise Beans %s. @Path annotations can only be placed on classes or interfaces that represent a local, remote or no-interface view of an Jakarta Enterprise Beans.")
+    @Message(id = 10, value = "Jakarta RESTful Web Services resource %s does not correspond to a view on the Jakarta Enterprise Beans %s. @Path annotations can only be placed on classes or interfaces that represent a local, remote or no-interface view of a Jakarta Enterprise Beans.")
```
Comment (bstansberry) [inline jaxrs/src/main/java/org/jboss/as/jaxrs/logging/JaxrsLogger.java:135]: "Bean"

File: testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/TopologyChangeListenerServlet.java [hunk 24]
```diff
@@ -18,7 +18,7 @@
 import jakarta.servlet.http.HttpServletResponse;
 
 /**
- * Utility servlet that delegates to an Jakarta Enterprise Beans to perform topology stabilization.
+ * Utility servlet that delegates to a Jakarta Enterprise Beans to perform topology stabilization.
```
Comment (bstansberry) [inline testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/TopologyChangeListenerServlet.java:21]: "Bean"

File: testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/securityapi/SecurityAPITestCase.java [hunk 25]
```diff
@@ -139,7 +139,7 @@ public void testCalls() throws Exception {
                     assertEquals("Unexpected content of HTTP response.", USERNAME, EntityUtils.toString(response.getEntity()));
                 }
 
-                // Verify a good username and password establishes an identity with the SecurityDomain within an Jakarta Enterprise Beans
+                // Verify a good username and password establishes an identity with the SecurityDomain within a Jakarta Enterprise Beans
```
Comment (bstansberry) [inline testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/securityapi/SecurityAPITestCase.java:142]: "Bean"

File: testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/securityapi/SecurityAPITestCase.java [hunk 26]
```diff
@@ -149,7 +149,7 @@ public void testCalls() throws Exception {
                     assertEquals("Unexpected content of HTTP response.", USERNAME, EntityUtils.toString(response.getEntity()));
                 }
 
-                // Verify a good username and password establishes an identity with the SecurityContext within an Jakarta Enterprise Beans
+                // Verify a good username and password establishes an identity with the SecurityContext within a Jakarta Enterprise Beans
```
Comment (bstansberry) [inline testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/securityapi/SecurityAPITestCase.java:152]: "Bean"

File: testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/jca/workmanager/distributed/ra/DistributedAdminObject1Impl.java [hunk 27]
```diff
@@ -20,7 +20,7 @@
  * That means that the tests that you write can't use
  * {@link org.jboss.arquillian.container.test.api.RunAsClient} and look up the object remotely.
  *
- * What you can do, is deploy an Jakarta Enterprise Beans, that will report the statistics via its admin object. You
+ * What you can do, is deploy a Jakarta Enterprise Beans, that will report the statistics via its admin object. You
```
Comment (bstansberry) [inline testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/jca/workmanager/distributed/ra/DistributedAdminObject1Impl.java:23]: "Bean"

File: testsuite/integration/microprofile/src/test/java/org/wildfly/test/integration/microprofile/jwt/ejb/BeanEndPoint.java [hunk 28]
```diff
@@ -18,7 +18,7 @@
 import org.eclipse.microprofile.jwt.JsonWebToken;
 
 /**
- * A simple Jakarta RESTful Web Services endpoint deployed as an Jakarta Enterprise Beans.
+ * A simple Jakarta RESTful Web Services endpoint deployed as a Jakarta Enterprise Beans.
```
Comment (bstansberry) [inline testsuite/integration/microprofile/src/test/java/org/wildfly/test/integration/microprofile/jwt/ejb/BeanEndPoint.java:21]: "Bean"

File: testsuite/integration/microprofile/src/test/java/org/wildfly/test/integration/microprofile/jwt/propagation/AnotherBeanEndPoint.java [hunk 29]
```diff
@@ -18,7 +18,7 @@
 import org.eclipse.microprofile.jwt.JsonWebToken;
 
 /**
- * A simple Jakarta RESTful Web Services endpoint deployed as an Jakarta Enterprise Beans.
+ * A simple Jakarta RESTful Web Services endpoint deployed as a Jakarta Enterprise Beans.
```
Comment (bstansberry) [inline testsuite/integration/microprofile/src/test/java/org/wildfly/test/integration/microprofile/jwt/propagation/AnotherBeanEndPoint.java:21]: "Bean"

File: testsuite/integration/microprofile/src/test/java/org/wildfly/test/integration/microprofile/jwt/propagation/BeanEndPoint.java [hunk 30]
```diff
@@ -18,7 +18,7 @@
 import org.eclipse.microprofile.jwt.JsonWebToken;
 
 /**
- * A simple Jakarta RESTful Web Services endpoint deployed as an Jakarta Enterprise Beans.
+ * A simple Jakarta RESTful Web Services endpoint deployed as a Jakarta Enterprise Beans.
```
Comment (bstansberry) [inline testsuite/integration/microprofile/src/test/java/org/wildfly/test/integration/microprofile/jwt/propagation/BeanEndPoint.java:21]: "Bean"

File: testsuite/integration/microprofile/src/test/java/org/wildfly/test/integration/microprofile/jwt/propagation/RemoteBeanEndPoint.java [hunk 31]
```diff
@@ -20,7 +20,7 @@
 import org.eclipse.microprofile.jwt.JsonWebToken;
 
 /**
- * A simple Jakarta RESTful Web Services endpoint deployed as an Jakarta Enterprise Beans.
+ * A simple Jakarta RESTful Web Services endpoint deployed as a Jakarta Enterprise Beans.
```
Comment (bstansberry) [inline testsuite/integration/microprofile/src/test/java/org/wildfly/test/integration/microprofile/jwt/propagation/RemoteBeanEndPoint.java:23]: "Bean"

File: testsuite/integration/microprofile/src/test/java/org/wildfly/test/integration/microprofile/jwt/propagation/sameVirtualDomain/RemoteBeanEndPoint.java [hunk 32]
```diff
@@ -20,7 +20,7 @@
 import org.eclipse.microprofile.jwt.JsonWebToken;
 
 /**
- * A simple Jakarta RESTful Web Services endpoint deployed as an Jakarta Enterprise Beans.
+ * A simple Jakarta RESTful Web Services endpoint deployed as a Jakarta Enterprise Beans.
```
Comment (bstansberry) [inline testsuite/integration/microprofile/src/test/java/org/wildfly/test/integration/microprofile/jwt/propagation/sameVirtualDomain/RemoteBeanEndPoint.java:23]: "Bean"

File: testsuite/shared/src/main/java/org/jboss/as/test/integration/web/sso/EJBServlet.java [hunk 33]
```diff
@@ -13,7 +13,7 @@
 import jakarta.servlet.http.HttpServletResponse;
 
 /**
- * A servlet that accesses an Jakarta Enterprise Beans and tests whether the call argument is
+ * A servlet that accesses a Jakarta Enterprise Beans and tests whether the call argument is
```
Comment (bstansberry) [inline testsuite/shared/src/main/java/org/jboss/as/test/integration/web/sso/EJBServlet.java:16]: "Bean"

File: undertow/src/main/java/org/wildfly/extension/undertow/deployment/WebComponentProcessor.java [hunk 34]
```diff
@@ -93,7 +93,7 @@ public void deploy(DeploymentPhaseContext phaseContext) throws DeploymentUnitPro
             ComponentDescription description = componentByClass.get(clazz);
             if (description != null) {
                 //for now just make sure it has a single view
-                //this will generally be a managed bean, but it could also be an Jakarta Enterprise Beans
+                //this will generally be a managed bean, but it could also be a Jakarta Enterprise Beans
```
Comment (bstansberry) [inline undertow/src/main/java/org/wildfly/extension/undertow/deployment/WebComponentProcessor.java:96]: "Bean"

**Comment by bstansberry** [comment]
@rhusar I should have done review style, but anyway, all of the above are just suggestions in case you want to do more that you have.  The only ones I'd really like to see are:

https://github.com/wildfly/wildfly/pull/18042/files#diff-10c2200d13a6415a8d02cc6af58700e9e59c8d0a234dcf3ebaf78fc9c7b081e0R2581

https://github.com/wildfly/wildfly/pull/18042/files#diff-4a8f75915fceb2759aad47b2bc0c1c9d328b4f58801cf757f6fe4f781bda7eaeR135

**Comment by bstansberry** [comment]
Also, if you do decide to fix all the detail items, as we discussed, using "Jakarta Enterprise Beans bean" instead of my proposed "Jakarta Enterprise Bean" is more correct; that latter is a made up term.

File: docs/src/main/asciidoc/_developer-guide/ejb3/Jakarta_Enterprise_Beans_Database_Persistent_Timers.adoc (hunk 1 above)
Comment (rhusar) [reply docs/src/main/asciidoc/_developer-guide/ejb3/Jakarta_Enterprise_Beans_Database_Persistent_Timers.adoc]: Went with former akin to "Jakarta Enterprise Beans bean" :-)

File: ejb3/src/main/java/org/jboss/as/ejb3/component/EJBValidationConfigurator.java (hunk 7 above)
Comment (rhusar) [reply ejb3/src/main/java/org/jboss/as/ejb3/component/EJBValidationConfigurator.java]: Fixed both

File: ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java (hunk 20 above)
Comment (rhusar) [reply ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java]: Uh... tried to make it a little better.

**Comment by rhusar** [comment]
@bstansberry OK, I went through everything manually with the intention to standardize on the spec being called "Jakarta Enterprise Beans" and a bean defined by that spec is a "Jakarta Enterprise Beans bean".

Just to note, there are visible changes to the log messages, to the domain model description strings and the documentation. So I would consider this a candidate for 33.x release.

**Review by bstansberry — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks @rhusar! I totally agree re having this in 33; that's why I jumped on doing the code review as soon as I saw what it was about.

**Pull request merged**